examples/Registration/paramFiles/aladin.par

```diff
@@ -0,0 +1,14 @@
+registration parameters - aladin :=
```

Comment:
I think rename the file to nifty_aladin.par or so? also for the other par files of course.